soroban-sdk/src/deploy.rs

```diff
@@ -141,6 +141,19 @@ impl Deployer {
             .update_current_contract_wasm(wasm_hash.into_val(&self.env).to_object())
             .unwrap_infallible();
     }
+
+    /// If the TTL for the provided contract is below `threshold` ledgers,
+    /// extend the TTL such that TTL == `extend_to`, where TTL is defined as
+    /// (live_until_ledger_seq for the entry - current ledger).
+    pub fn extend_ttl(&self, contract_address: Address, threshold: u32, extend_to: u32) {
```

Comment:
Suggested reword of the doc to lead with a short single statement that says what the fn does, that then follows with further discussion about exact behavior, and then finishes with contextual information.

```suggestion
    /// Extend the TTL of the contract instance.
    ///
    /// Extends the TTL only if the TTL for the provided contract is below `threshold` ledgers.
    ///
    /// The TTL is the number of ledgers between the current ledger and the `live_until_ledger_seq` value for the ledger entry.
    pub fn extend_ttl(&self, contract_address: Address, threshold: u32, extend_to: u32) {
```